config/purposes/pWGS_384_bottom.yml

```diff
@@ -0,0 +1,26 @@
+---
+pWGS-384 Post Shear XP:
+  :asset_type: plate
+  :creator_class: LabwareCreators::QuadrantStamp
+  :merger_plate: true
+  :size: 384
+pWGS-384 End Prep:
+  :asset_type: plate
+  :presenter_class: Presenters::PermissivePresenter
```

Comment:
Comment in class says: 
```ruby
  # Class PermissivePresenter provides a presenter which allows plate creation
  # even when the plate is pending
```
However the most similar plate to this is LHR-384 End Prep: which it also has this same class. I assume there is a reason why is needed. It would be worth to investigate in which situation is required, just in case we are using something we shouldn't be.

Reply:
Yes there is a reason this was added. As part of the workflow, the pWGS-384 AL Lib plate needs to be created from the pWGS-384 End Prep plate before that plate has `passe`d. (Both the AL Lib and End Prep plates are `started` in the same robot step, and `passed` in subsequent steps on the same robot). As the comment suggests, including this presenter class allows creating the AL Lib plate from the End Prep plate despite it not being `passed`